modules/administration-guide/partials/con_che-operator.adoc

```diff
@@ -17,7 +17,10 @@ Creates and controls the necessary {orch-name} objects to run a {prod-short} ins
 `CheCluster` custom resource (CR)::
 On a cluster with the {prod-short} operator, it is possible to create a `CheCluster` custom resource (CR). The {prod-short} operator ensure full lifecycle management of the {prod-short} server components on this {prod-short} instance.
 
+The {prod-short} operator is also provides routing for {devworkspace} by configuring the `che-gateway`.
```

Comment:
Does it mean the {prod-short} operator creates an instance of the `che-gateway` before you create a `CheCluster` CR? I thought the `che-gateway` was part of a `CheCluster` CR deployment.

Reply:
I don't understand your question. che-operator deploys and configure che-gateway. I've put this sentence here, because che-operator is now part of devworkspace startup process where it provides routing for the devworkspace. It does so by configuring the che-gateway.

Reply:
I think that what @themr0c means is that we do not list the server side components controlled by the operator here. Something that you could do is change the description of the `{prod-short} controller::` above and add that beyond {orch-name} objects it also controls a dedicated gateway that routes inbond and outbound traffic to workspaces.

Reply:
I believe that:

When you install Che operator, you get:

* CheCluster CRD

When you create a CheCluster CR, you get:

* Che server side services, including Che gateway and Devworkspace operator

And not: 

When you install Che operator, you get:

* CheCluster CRD
* Che gateway
* Devworkspace operator

When you create a CheCluster CR, you get:

* Che server side services

Reply:
ah, I think I'm getting closer to understand what you mean. Yes, you need to create `CheCluster` CR instance to get all che deployment (che-server, che-gateway, registries ... I'm not 100% sure about devworkspace-operator).

What I wanted to emphasize here is that che-operator is now not only responsible for Che deployment, but it also touches workspace startup process by configuring routing for workspaces.